management/server/store/sql_store.go

```diff
@@ -1775,33 +1857,53 @@ func (s *SqlStore) GetGroupsByIDs(ctx context.Context, lockStrength LockingStren
 	}
 
 	var groups []*types.Group
-	result := tx.Find(&groups, accountAndIDsQueryCondition, accountID, groupIDs)
+	result := tx.Preload(clause.Associations).Find(&groups, accountAndIDsQueryCondition, accountID, groupIDs)
```

Comment:
@pascal-fischer I think we need to be careful with `Preload`, I remember having issues before, where instead of loading only the associations for the FK of the type, it was loading everything. Please debug these queries and if needed specify a join entry.